state/runtime/precompiled/modexp.go

```diff
@@ -118,10 +122,43 @@ func (m *modExp) gas(input []byte, config *chain.ForksInTime) uint64 {
 		gasCost.Set(baseLen)
 	}
 
+	//EIP-2565 gas cost calculation
+	if config.EIP2565 {
+		// EIP-2565 has three changes
+		// 1. Different multComplexity (inlined here)
+		// in EIP-2565 (https://eips.ethereum.org/EIPS/eip-2565):
+		//
+		// def mult_complexity(x):
+		//    ceiling(x/8)^2
+		//
+		//where is x is max(length_of_MODULUS, length_of_BASE)
+		gasCost.Add(gasCost, big7)
+		gasCost.Div(gasCost, big8)
+		gasCost.Mul(gasCost, gasCost)
+
+		// a = a * max(ADJUSTED_EXPONENT_LENGTH, 1)
+		if adjExpLen.Cmp(big1) >= 0 {
+			gasCost.Mul(gasCost, adjExpLen)
+		} else {
+			gasCost.Mul(gasCost, big1)
+		}
+		// 2. Different divisor (`GQUADDIVISOR`) (3)
+		gasCost.Div(gasCost, big3)
+		// cap to the max uint64
+		if !gasCost.IsUint64() {
+			return math.MaxUint64
+		}
+		// 3. Minimum price of 200 gas
+		if gasCost.Uint64() < 200 {
```

Comment:
Nitpick: consider extracting 200 to some constant?

Reply:
we can but Geth hasn't extracted the value to a constant.

Reply:
Your call. However, we don't need to copy the entire Geth codebase in order to comply with Ethereum specification

Generally speaking, the good practice is not relying on magic numbers, but instead capturing them into constants, which brings additional semantics and better code readability.